src/lazyLoad/nuxt_js/nuxt_getFromPageSource.ts

```
@@ -0,0 +1,82 @@
+        }
+    }
+
+    // now, search all the script tags
+    const scriptTags = $("script");
+    for (const scriptTag of scriptTags) {
+        const src = $(scriptTag).attr("src");
+        if (
+            src !== undefined &&
+            src.match(
+                /(https:\/\/[a-zA-Z0-9_\_\.]+\/.+\.js\??.*|\/.+\.js\??.*)/
+            )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)